.github/workflows/issue-triage-runner.yml

```
@@ -0,0 +1,81 @@
+          ISSUE_NUMBER:    ${{ github.event.issue.number }}
+          ISSUE_TITLE:     ${{ github.event.issue.title }}
+          ISSUE_BODY:      ${{ github.event.issue.body }}
+          ISSUE_LABELS:    ${{ join(github.event.issue.labels.*.name, ',') }}
+          ISSUE_URL:       ${{ github.event.issue.html_url }}
+          ISSUE_ACTION:    ${{ github.event.action }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+        run: |
+          set -euo pipefail
+          result=$(bash scripts/aw.sh run issue-triage --staged)
+          skip=$(echo "$result" | python3 - <<'PYEOF'
+import json, sys
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)